src/cljdoc/server/build_log.clj

```diff
@@ -62,8 +62,11 @@
                  ["id = ?" build-id]))
   (get-build [_ build-id]
     (first (sql/query db-spec ["SELECT * FROM builds WHERE id = ?" build-id])))
-  (recent-builds [_ limit]
-    (sql/query db-spec ["SELECT * FROM builds ORDER BY id DESC LIMIT ?" limit]))
+  (recent-builds [_ days]
+    (sql/query db-spec [(str "SELECT * FROM builds "
+                             "WHERE analysis_triggered_ts "
+                             (format "BETWEEN DATETIME('now', 'localtime', '-%s days')" days)
+                             "AND DATETIME('now', 'localtime', '+1 days');")]))
```

Comment:
Oh yeah, I tried to use `?` but it failed with a cryptic error message that I couldn't make sense of : 
```
1. Unhandled java.lang.ArrayIndexOutOfBoundsException
   0
```
So I changed to `format`. Any clue?

Reply:
No clue right now but I'll take a closer look at it later 👍 Must be possible somehow, right? 😄

Reply:
Ok. I still don't know if you can just get the plain `days` to be placed into the string but I think this approach should work:

```clojure
;; slightly shortened version of the query
(sql/query db-spec ["SELECT * FROM builds WHERE analysis_triggered_ts = DATETIME('now', 'localtime', ?)" (str "-" days " days")])
```

By supplying one string as parameter the preprocesser can catch any problematic SQL injection stuff which would otherwise not be done properly.

Let me know if this makes sense 🙂

Reply:
Bingo! This worked.

I think I found my mistake. Earlier was trying something like which fails:
```
(sql/query db-spec ["SELECT * FROM builds WHERE analysis_triggered_ts = DATETIME('now', 'localtime', '-? days')" 30])
```
> By supplying one string as parameter the preprocesser can catch any problematic SQL injection stuff which would otherwise not be done properly.

Makes sense. I should have thought about it before. 
Thanks for point out 🙂